Doppler.Sap/Controllers/BusinessPartnerController.cs

```diff
@@ -50,9 +57,9 @@ public async Task<IActionResult> CreateOrUpdateBusinessPartner([FromBody] Dopple
 
         private string VerifyUserInformation(DopplerUserDto dopplerUser)
         {
-            if (dopplerUser.BillingCountryCode != "AR")
+            if (!_sapConfig.SapServiceConfigs.Any(c => c.CountryCode == dopplerUser.BillingCountryCode))
             {
-                _logger.LogInformation($"{dopplerUser.Email} won't be sent to SAP because it's not from AR");
+                _logger.LogInformation($"{dopplerUser.Email} won't be sent to SAP because it's not from AR or US");
```

Comment:
```suggestion
                _logger.LogInformation($"{dopplerUser.Email} won't be sent to SAP because it's not from {string.Join(", ", _sapConfig.SapServiceConfigs.Select(x => x.CountryCode))}");
```